src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1527,6 +1527,58 @@ internal bool IsRealSigned
             Predicate<ISymbol> filterOpt,
             CancellationToken cancellationToken);
 
+        internal bool StartSourceChecksumCalculation(DebugDocumentsBuilder documentsBuilder, DiagnosticBag diagnostics)
```

Comment:
>StartSourceChecksumCalculation [](start = 22, length = 30)

CalculateSourceChecksum?  In general I expect Start methods to have End pairs.  Not the case here so we should choose a more representative name.